src/lava/magma/core/model/model.py

```diff
@@ -6,7 +6,7 @@
 
 import typing as ty
 import logging
-from abc import ABC
+from abc import ABC, abstractmethod
```

Comment:
abstractmethod imported but not used in function declaration

Reply:
deleted.